### DISCUSSION TITLE
non-geospatial tilelayer moon example question

### DISCUSSION THREAD

**Discussion by cornhundred** [visgl/deck.gl#5676] [Q&A] [answered]
Hi, I have a question about the non-geospatial tilelayer moon example https://deck.gl/examples/tile-layer-non-geospatial/

how are you all arriving at the `15` as the offset used in the deck.gl zoom to deepzoom zoom level [code here](https://github.com/visgl/deck.gl/blob/8.4-release/examples/website/image-tile/app.js#L63)?

```
      getTileData: ({x, y, z}) => {
        return load(`${ROOT_URL}/moon.image_files/${15 + z}/${x}_${y}.jpeg`);
      },
```

**Comment by ilan-gold** [comment, marked as answer]
There are 16 layers of the pyramid indexed from 0-15 in the inverted order of deck.gl's zoom.  So if you want 0 zoom in deck.gl to be the highest resolution, it needs to go to 15 in the deep zoom pyramid, -1 next highest to 14 in the deep zoom, and so forth.  See here: https://github.com/visgl/deck.gl-data/tree/master/website/image-tiles/moon.image/moon.image_files
